randovania/games/prime1/gui/dialog/prime_cosmetic_patches_dialog.py

```diff
@@ -37,6 +85,48 @@ def persist(value: int):
 
         return persist
 
+    def _persist_suit_color_rotations(self):
+        self._cosmetic_patches = dataclasses.replace(
+            self._cosmetic_patches,
+            **{'suit_color_rotations': (
```

Comment:
Replace the `**{'suit_color_rotations':` with just `suit_color_rotations=`. The `**dict` trick is good when the field name is variable somehow.